tests/integration/tier0/latest-kernel-check/test_latest_kernel_check.py

```diff
@@ -14,7 +14,7 @@ def tainted_repository(shell):
     repofile = "broken_repo"
     centos_custom_reposdir = "/usr/share/convert2rhel/repos/"
 
-    # For CentOS we are working with hardcoded repos in /usr/share/convert2rhel/repos/centos-8.{4,5}
+    # For CentOS, we are working with hardcoded repos in /usr/share/convert2rhel/repos/centos-8.{4,5}
```

Comment:
the 8.4 are removed no?
```suggestion
    # For CentOS, we are working with hardcoded repos in /usr/share/convert2rhel/repos/centos-8.5
```

Reply:
Thanks for spotting this